### PR TITLE
Add amountChargedBack to getPayment schema

### PR DESCRIPTION
**Pull request by maicolgarces**
Add `amountChargedBack` to `getPayment response`.

### CODE REVIEW

File: source/reference/v2/payments-api/get-payment.rst [hunk 1]
```diff
@@ -242,6 +242,30 @@ Response
...
+
+              .. type:: string
+
+            - The `ISO 4217 <https://en.wikipedia.org/wiki/ISO_4217>`_ currency code.
+
+          * - ``value``
+
+              .. type:: string
+
+            - A string containing the exact captured amount of the payment in the given currency.
```
Comment (ndijkstra) [inline source/reference/v2/payments-api/get-payment.rst:266]: This line is still talking about captures ;p

**Review by ndijkstra — CHANGES_REQUESTED** [review]
Can you also add it to the v2 changelog?

**Comment by sandervanhooft** [comment]
🤩

File: source/reference/v2/payments-api/get-payment.rst (hunk 1 above)
Comment (maicolgarces) [reply source/reference/v2/payments-api/get-payment.rst]: 🤦 fixed

**Pull request merged**